dissect/target/plugins/os/windows/defender.py

```diff
@@ -4,7 +4,7 @@
 from datetime import datetime, timezone
 from io import BytesIO
 from pathlib import Path
-from typing import Any, BinaryIO, Generator, Iterable, Iterator, TextIO, Union
+from typing import Any, BinaryIO, Iterable, Iterator, TextIO, Union
```

Comment:
```suggestion
from typing import Any, BinaryIO, Iterable, Iterator, TextIO
```
Could you remove the `Union` from this file too?